overview/index.markdown

```diff
@@ -0,0 +1,55 @@
+---
```

Comment:
Is there a reason this is in a subdirectory, and not a top-level file (i.e. `overview.markdown`)?
The [blog](https://github.com/kroxylicious/kroxylicious.github.io/blob/main/blog.markdown) and [index](https://github.com/kroxylicious/kroxylicious.github.io/blob/main/index.markdown) pages are both top level, and though the [quickstarts](https://github.com/kroxylicious/kroxylicious.github.io/tree/main/quickstarts) page doesn't follow this convention, that's done on purpose so that we can keep all of that content together in one place. It doesn't make sense to me for this to be on its own in a dedicated directory.

Reply:
The idea was that the overview might need more than a single page.  That's what motivated me to use a directory.
I'd be happy to use a top level page and we could break it out later, once the actual need arose.

Reply:
Done.